### ISSUE TITLE
Closing drawing mode with ESC key doesn't work

### ISSUE THREAD

**Issue by ceskyDJ** [alyssaxuu/screenity#209]
Hi @alyssaxuu! I tried to exit drawing mode via ESC key, but it doesn't work. It just closes the information bubble that says that you can close it using ESC key. When I click on the information bubble, it closes drawing mode correctly. I took a video of the problem:

https://github.com/alyssaxuu/screenity/assets/48548230/30a4f6cc-dcda-4dcf-a9e1-455e1a9a7b3e

**SW details:**

- **OS:** Debian 11
- **Browser:** Google Chrome Version 120.0.6099.129 (Official Build) (64-bit)
- **Screenity:** 3.1.1 (installed manually)

**Comment by alyssaxuu** [comment]
Just checked this and I was able to reproduce the issue. Looks like I wasn't using the proper event handler from the Radix UI Toast component. 

I've just fixed it and added it to the next release, but I'm still looking to add a few varied improvements as part of the release so it could be a couple days before I publish it (and the version 3.1.1 is stuck on review in the Chrome Store anyway so it might take some time).
